src/daft-sql/src/functions.rs

```diff
@@ -242,14 +243,21 @@ impl<'a> SQLPlanner<'a> {
         // assert using only supported features
         check_features(func)?;
 
+        fn get(
```

Comment:
rename this to something more descriptive

Reply:
Updated.